src/views/refund/refund.js

```diff
@@ -24,6 +24,18 @@ const styles = theme => ({
   },
 });
 
+const uploadRefundFileText = (refundFile, txHash) => {
+  const hasRefundFile = Object.keys(refundFile).length === 0;
```

Comment:
The name of the variable is a little confusing because it is `true` if there is no refund file